andela/ah-frontend-thanos
# #161348764 Share articles on social media.

#### What does this PR do?
A user can share an article on `Facebook`, `Twitter`,` GooglePlus`, `LinkedIn`


#### Description of Task to be completed?
- Users can share articles on any of the social media platforms available


#### How should this be manually tested?
- Go to the article's URL: `/article/:articleId` e.g. `/article/4` 
- You should be able to see the social media icons on the article
- Click on any of the icons and you can then share

#### Any background context you want to provide?
You do not  need to be logged in to add a tag to an article
#### What are the relevant pivotal tracker stories?
[#161348764](https://www.pivotaltracker.com/story/show/#161348764

#### Relevant Screenshots
<img width="1458" alt="screen shot 2018-12-17 at 8 10 53 pm" src="https://user-images.githubusercontent.com/22786444/50103145-0bbef980-0238-11e9-8256-ac96162c3e88.png">
<img width="1035" alt="screen shot 2018-12-17 at 8 11 07 pm" src="https://user-images.githubusercontent.com/22786444/50103160-111c4400-0238-11e9-906b-01e8a13a3411.png">
<img width="663" alt="screen shot 2018-12-17 at 8 11 43 pm" src="https://user-images.githubusercontent.com/22786444/50103171-19747f00-0238-11e9-9366-5f40ee11dc8d.png">
